articles/chaos-studio/chaos-studio-faults-actions.md

```diff
@@ -29,7 +29,7 @@ There are two varieties of actions in Chaos Studio:
 
 Faults are the most common action in Chaos Studio. Faults cause a disruption in a system, allowing you to verify that the system effectively handles that disruption without impacting availability. Faults can be destructive (for example, killing a process), apply pressure (for example, adding virtual memory pressure), add latency, or cause a configuration change. In addition to a name and type, faults may also have a *duration*, if continuous, and *parameters*. Parameters describe how the fault should be applied and are specific to the fault name. For example, a parameter for the Cosmos DB failover fault is the read region that will be promoted to the write region during the write region failure. Some parameters are required while others are optional.
 
-Faults are either *agent-based* or *service-direct* depending on the target type. An agent-based fault requires the Chaos Studio agent to be installed on a virtual machine or virtual machine scale set. The agent is available for both Windows and Linux, but not all faults are available on both operating systems. See the [fault library](chaos-studio-fault-library.md) for information on which faults are supported on each operating system. Service-direct faults do not require any agent - they run directly against an Azure resource.
+Faults are either *agent-based* or *service-direct* depending on the target type. An agent-based fault requires the Chaos Studio agent to be installed on a virtual machine or virtual machine scale set. The agent is available for both Windows and Linux, but not all faults are available on both operating systems. At present multiple agent-based faults cannot be run. See the [fault library](chaos-studio-fault-library.md) for information on which faults are supported on each operating system. Service-direct faults do not require any agent - they run directly against an Azure resource.
```

Comment:
@prasha-microsoft - the language here is unclear about the fact that this means _at the same time_ and _on the same machines_ . Please update the text to something like, "At present, only one agent-based fault can be run per machine at a time."